docs/api/web-contents.md

```diff
@@ -917,19 +917,26 @@ For the `mouseWheel` event, the `event` object also have following properties:
 * `hasPreciseScrollingDeltas` Boolean
 * `canScroll` Boolean
 
-### `webContents.beginFrameSubscription(callback)`
+### `webContents.beginFrameSubscription(callback, onlyDamaged)`
```

Comment:
Current convention for APIs is to put callback as last parameter, can you make the signature of this API look like below?

```js
beginFrameSubscription([onlyDamaged, ]callback)
```